gatsby-node.js

```diff
@@ -28,7 +28,7 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
       if (Object.prototype.hasOwnProperty.call(node.frontmatter, "slug"))
         slug = `/${kebabCase(node.frontmatter.slug)}`;
       if (Object.prototype.hasOwnProperty.call(node.frontmatter, "date")) {
-        const date = moment(node.frontmatter.date, siteConfig.dateFromFormat);
+        const date = moment.utc(node.frontmatter.date);
```

Comment:
`moment` is pretty smart about parsing dates, no need to give the format explicitly (especially since we did not check it anywhere :see_no_evil: )

Also, assume the dates to be in UTC to avoid shifts by some hours depending on the system locale the website is built on.